packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -489,10 +489,12 @@ export async function deleteFolder(
     deleteParams.Delete.Objects.push({ Key: content.Key })
   })
 
-  const deleteResponse = await client.deleteObjects(deleteParams)
-  // can only empty 1000 items at once
-  if (deleteResponse.Deleted?.length === 1000) {
-    return deleteFolder(bucketName, folder)
+  if (deleteParams.Delete.Objects.length) {
```

Comment:
Does this mean we will leave empty buckets? Is this expected?

Reply:
Its a single bucket used for all apps - so if we didn't add anything theres simply nothing to do - we use static buckets for everything, so deleting them would make it harder to create new things.